WatchDogIntelliJPlugin/WatchDog/resources/META-INF/plugin.xml

```diff
@@ -1,7 +1,7 @@
 <idea-plugin version="2">
     <id>nl.tudelft.watchdog</id>
     <name>WatchDog</name>
-    <version>1.5.0</version>
+    <version>1.5.0.1</version>
```

Comment:
Okay, I'll update here, but I will leave updating the version across all projects (Core/Eclipse/IntelliJ/Server) for Tuesday, there are now even more places to update than before :)